zbench.zig

```diff
@@ -89,6 +89,96 @@ pub const Benchmark = struct {
         });
     }
 
+    /// An incremental API for getting progress updates on running benchmarks.
+    pub const Iterator = struct {
+        allocator: std.mem.Allocator,
+        b: *const Benchmark,
+        remaining: []const Definition,
+        runner: ?Runner,
+
+        /// A summary of progress through benchmarking a function. The
+        /// total_runs field may vary over time as the runner calibrates how
+        /// many runs to perform.
+        pub const Progress = struct {
```

Comment:
What do you think about using the std lib for displaying the progress?
https://ziglang.org/documentation/master/std/#std.Progress

Used it in [another project](https://github.com/hendriknielaender/zvm/pull/22/files), came in handy.

Reply:
Nice suggestion, it's pretty easy to use from following the example, I've added it to the sample code.

Reply:
Cool 👍